vertica_python/tests/integration_tests/test_cursor.py

```diff
@@ -382,6 +382,15 @@ def test_format_quote_unicode(self):
             formatted_word = u''.join((u'"', re.escape(bad_word), u'"'))
             self.assertEqual(formatted_word, cur.format_quote(bad_word, True))
 
+    def test_execute_parameter(self):
+        with self._connect() as conn:
+            cur = conn.cursor()
+            val = u"".join(chr(i) for i in range(1, 128))
+            cur.execute("SELECT :val",
+                        parameters={"val": val},
+                        use_prepared_statements=False)
+            self.assertEqual([val], cur.fetchone())
+
```

Comment:
I suggest this test to be defined under `SimpleQueryTestCase` class. `CursorTestCase` class contains tests that should be run for both use_prepared_statements=True/False. This change only affects use_prepared_statements=False behaviors and `SimpleQueryTestCase` class is the right place for tests that can run for only use_prepared_statements=False.

Thanks for reminding me there are confusions about test classes, I'll add comments to them.

Reply:
Yes, I was confused by that, moved it to its proper place.